deploy/templates/nstemplatetiers/basic/ns_code.yaml

```diff
@@ -34,27 +34,27 @@ objects:
 - apiVersion: rbac.authorization.k8s.io/v1
   kind: RoleBinding
   metadata:
-    name: user-edit
+    name: user-rbac-edit
     namespace: ${USERNAME}-code
   roleRef:
     apiGroup: rbac.authorization.k8s.io
-    kind: ClusterRole
-    name: edit
+    kind: Role
+    name: rbac-edit
   subjects:
-  - kind: User
-    name: ${USERNAME}
+    - kind: User
```

Comment:
Is the spacing here correct? I think you removed the extra spaces from the other subjects.

Reply:
I guess it's just how my IDE formatted it. Both versions should work. And tests are passing, so..